hilti/toolchain/src/compiler/printer.cc

```diff
@@ -35,15 +35,20 @@ namespace {
 struct Printer : visitor::PreOrder {
     Printer(printer::Stream& out) : _out(out) {} // NOLINT
 
-    void printFunctionType(const type::Function& ftype, const ID& id) {
+    void printFunctionType(const type::Function& ftype, const ID& id, function::CallingConvention calling_conv) {
         if ( ftype.isWildcard() ) {
             _out << "<function>";
             return;
         }
 
-        if ( ftype.flavor() != type::function::Flavor::Standard )
+        if ( ftype.flavor() == type::function::Flavor::Standard )
+            _out << "function ";
+        else
             _out << to_string(ftype.flavor()) << ' ';
```

Comment:
If we renamed `Flavor::Standard` to `Flavor::Function` (along with a corresponding string mapping) we could save this special-case here.